src/api/environments.ts

```diff
@@ -14,18 +14,6 @@ export const sdkTypes = [
     'server'
 ]
 
-export class Environment {
-    _id: string
-    _project?: string
-    name?: string
-    key: string
-    type: 'development' | 'staging' | 'production' | 'disaster_recovery'
-    sdkKeys: Record<string, APIKey[]>
-    description?: string
-    createdAt: Date
-    updatedAt: Date
-}
-
 export class CreateEnvironmentParams {
```

Comment:
should we be replacing these `CreateEnvironmentParams`, `CreateProjectParams`, etc. with the types in the zodClient instead? like `CreateEnvironmentDto` and `CreateProjectDto`?

Reply:
made a ticket to handle this! https://taplytics.atlassian.net/browse/DVC-7720